web-common/src/features/templates/image/Image.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import { ImageProperties } from "@rilldata/web-common/features/templates/types";
+  import { V1ComponentSpecRendererProperties } from "@rilldata/web-common/runtime-client";
+
+  export let rendererProperties: V1ComponentSpecRendererProperties;
+
+  // Define default values for image properties
+  const defaultImageProperties: ImageProperties = {
+    url: "",
+    size: "contain",
+    adjust: {
+      opacity: 1,
+      blur: 0,
+      saturation: 1,
```

Comment:
We can also have something like this 

``` 
 interface ImageProperties {
    url: string;
    style?: Partial<CSSStyleDeclaration>;
  }

  const DEFAULT_IMAGE_PROPERTIES: ImageProperties = {
    url: "",
    style: {
      objectFit: "contain",
      opacity: "1",
      filter: "blur(0px) saturate(1)",
    },
  };
```
This way users can mention any style possible.

Reply:
`imageProperties` then can be updated as 

```
  $: imageProperties = {
    ...DEFAULT_IMAGE_PROPERTIES,
    ...rendererProperties,
    style: {
      ...DEFAULT_IMAGE_PROPERTIES.style,
      ...rendererProperties.style,
    },
  } as ImageProperties;

  $: imageStyle = Object.entries(imageProperties.style || {})
    .map(([key, value]) => `${kebabCase(key)}: ${value};`)
    .join(' ');
```